README.md

```diff
@@ -15,51 +15,105 @@ When you are done:
 #### Loops & Lists
 - [Max Value](https://repl.it/@Admin7/maxvalue)
 ```python
-# copy & paste your solution here
+def return_max(l):
+  if len(l)== 0:
+    return None
+  return max(l)
+def return_min(l):
+  if len(l)== 0:
+    return None
+  return min(l)
 ```
 
 - [Min Value](https://repl.it/@Admin7/minvalue)
 ```python
-# copy & paste your solution here
+def return_min(l):
+  if len(l)== 0:
+    return None
+  return min(l)
 ```
 
 - [Extend](https://repl.it/@Admin7/extendlist)
 ```python
-# copy & paste your solution here
+for i in iter:
+    l.append(i)
```

Comment:
You can also you the `extend` list method to append an iterable to the end of an array without looping.